packages/store/src/internal/internals.ts

```diff
@@ -65,6 +65,13 @@ export interface MappedStore {
   depth: string;
 }
 
+export type StatesAndDefaults =
+  | {
```

Comment:
Is this first `|` supposed to be here? There is nothing before it.

Reply:
@markwhitfeld This is prettier :(